### PR TITLE
withonline.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/120750

<details><summary>Screenshot:</summary>

Снимок экрана 2022-06-06 в 15 42 59

</details><br/>

### CODE REVIEW

**Review by Yuki2718 — COMMENTED** [review]
Probably we should replace
```
withonline.jp##div[id$="_inread"]
withonline.jp##div[id$="_InArticle-1"]
```
with `withonline.jp##div[id^="TW_"]`. Not sure if the extended rule is still needed then (on PC covered by `withonline.jp##div[style="max-width:970px;min-height:90px;text-align:center;margin:0 auto;"]`).

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -100,6 +100,7 @@ otonasalone.jp,sumaiweb.jp#?#._popIn_infinite_page > a._popIn_recommend_article_
 !
 !
 avgle.net#?#.column > div.card > div.adv:upward(2)
+withonline.jp#?##TW_PC_Billboard:upward(1)
```
Comment (Alex-302) [inline JapaneseFilter/sections/css_extended.txt:103]: Можно обычным правилом, т.к. 970*90 - часто используемый размер баннеров
<details><summary>Screenshot:</summary>

image

</details>

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/css_extended.txt (hunk 1 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/css_extended.txt:103]: We already have `withonline.jp##div[style="max-width:970px;min-height:90px;text-align:center;margin:0 auto;"]`.

Comment (zloyden) [inline JapaneseFilter/sections/css_extended.txt:103]: Actually, we do not need the rule as @Yuki2718 said.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — COMMENTED** [review]
> Probably we should replace

According to the their code, it only affects the leftovers. I have changed the rules.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -63,6 +63,7 @@ note.calbee.jp,note.linemusic.jp,diehardtales.com,note.com,note.kagg.jp,note.jp,
...
 !
 veryweb.jp##.c-articleShare
+withonline.jp##.pattern_other > div.other-box:has(> div.heading-area > p:contains(SHARE))
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:66]: Check this
```
withonline.jp##.pattern_other > div.other-box:has(> ul[class^="share-"])
```

File: SocialFilter/sections/specific.txt [hunk 3]
```diff
@@ -62,6 +62,8 @@ note.calbee.jp,note.linemusic.jp,diehardtales.com,note.com,note.kagg.jp,note.jp,
...
 !
+veryweb.jp##.c-articleShare
+withonline.jp##.pattern_other > div.other-box:has(> ul[class^="share-"])
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:66]:
```
withonline.jp#?#.pattern_other > div.other-box:has(> ul[class^="share-"])
```
Should be in `css_extended`

**Review by zloyden — CHANGES_REQUESTED** [review]

**Pull request closed without merging**